src/components/cardbuilder/card.css

```diff
@@ -157,7 +157,7 @@ button::-moz-focus-inner {
 }
 
 .cardImageContainer {
-    background-size: cover;
+    background-size: contain;
```

Comment:
This will introduce that change for all cards.

The fix needs to be more specific, to only target the cards used in the Edit Images dialog.

Reply:
Will do. I have reverted the above changed and moveed the css into the two js files